src/renderer/components/layout/sidebar.tsx

```diff
@@ -131,125 +132,123 @@ export class Sidebar extends React.Component<Props> {
   }
 
   render() {
-    const { toggle, isPinned, className } = this.props;
+    const { toggle, compact, className } = this.props;
```

Comment:
Instead of direct access from `sidebarLocalStorage` you mean? To make `Sidebar` less dependent and testable + `compact` also used in `MainLayout` where it comes from.

Reply:
Sounds good.